tests/CMakeLists.txt

```diff
@@ -241,6 +241,10 @@ add_test(NAME plugin_http_api_test COMMAND tests/plugin_http_api_test.py WORKING
 set_tests_properties(plugin_http_api_test PROPERTIES TIMEOUT 50)
 set_property(TEST plugin_http_api_test PROPERTY LABELS nonparallelizable_tests)
 
+add_test(NAME plugin_http_category_api_test COMMAND tests/plugin_http_api_test.py WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+set_tests_properties(plugin_http_category_api_test PROPERTIES TIMEOUT 50 ENVIRONMENT "PLUGIN_HTTP_TEST_CATEGORY=ON")
+set_property(TEST plugin_http_category_api_test PROPERTY LABELS nonparallelizable_tests)
```

Comment:
How is different then the existing `plugin_http_api_test`, no new args are passed to the test.

Reply:
I see this passes an ENVIRONMENT variable. Can we change that to an argument to the python test.